### ISSUE TITLE
Spirit Combat Damage shows when it should not, but only once

### ISSUE THREAD

**Issue by Moonpile** [sun-dragon-cult/fvtt-system-rqg#488]
### _How:_ To Reproduce

1. Import the Disease Spirit from the Bestiary
2. Open it and see the Spirit Combat Damage (that should not be there)
3. Close it
4. Open it again and see that the Spirit Combat Damage is not visible (correct)
5. 
### Expected behavior

You should never see the Spirit Combat Damage on this creature

### Screenshots

Upon first opening after importing:
![image](https://user-images.githubusercontent.com/3959760/221094189-19d4b6c9-ca3b-4ff5-af30-b0f3dda61965.png)

After closing and re-opening:
![image](https://user-images.githubusercontent.com/3959760/221094258-36ecf428-aedd-4fae-acf4-f146a157a6df.png)

### Environment - please paste the information from foundry settings tab "support" button:

2.1.0

### Additional context

n/a

**Comment by Moonpile** [comment]
I don't think this one needs to go in 2.1.1. It's not an actual problem, just cosmetic.

**Comment by wake42** [comment]
Fixed on branch `fix/spirit-magic-cha-limit` (commit 26ec47df).

Root cause: the health/hit-location UI section and `getCombinedActorHealth()` both keyed visibility/status purely off `system.attributes.hitPoints.max`. For actors with no physical characteristics (like the Disease Spirit) this value is `null` before the first derived-data pass runs, so on first render after import the health section showed up; on the second open, derived data had already run once and correctly hid it.

Fix: both checks now explicitly look for STR/CON/SIZ characteristics or a Hit Location item to decide whether an actor tracks hit points at all, instead of relying on the possibly-stale `hitPoints.max`. Actors that don't track hit points (spirits) now consistently hide the health/hit-location section, and their health status is instead derived from magic points — they go "unconscious" when magic points hit 0, same rule already used for normal actors, rather than incorrectly showing hit-point-based wounds/death.
